src/utils/dato-utils.ts

```diff
@@ -234,3 +234,31 @@ export function hentSkjermetInfo(
         };
     }
 }
+
+export function hentSikkerhetsTiltakInfo(
+    harSikkerhetsTiltak: boolean | undefined,
+    tiltakBeskrivelse: string | undefined,
+    tiltakFra: string | undefined,
+    tiltakTil: string | undefined
+): SikkerhetsTiltakEtikettConfig {
+    if (!harSikkerhetsTiltak) {
+        return {
+            hidden: true,
+            tittel: null,
+            type: 'info'
+        };
+    }
+
+    var tittelVerdi = 'Sikkerhetstiltak';
+    if (tiltakBeskrivelse && tiltakBeskrivelse.length > 0) {
+        tittelVerdi += ': ' + tiltakBeskrivelse;
+    }
+    if (tiltakFra && tiltakTil) {
```

Comment:
Why are we checking `tiltakFra` when only using `tiltakTil`?

Reply:
first idea was to show also tiltakFra...I will remove it